API.md

```diff
@@ -122,17 +122,50 @@ script loaders are also supported.
 </html>
 ```
 
-The Open MCT library included above requires certain assets such as html 
-templates, images, and css. If you installed Open MCT from GitHub as described 
+The Open MCT library included above requires certain assets such as html
+templates, images, and css. If you installed Open MCT from GitHub as described
 in the section on [Building from Source](#building-from-source) then these
-assets will have been downloaded along with the Open MCT javascript library. 
+assets will have been downloaded along with the Open MCT javascript library.
 
-There are some plugins bundled with the application that provide UI, 
-persistence, and other default configuration which are necessary to be able to 
-do anything with the application initially. Any of these plugins can, in 
-principle, be replaced with a custom plugin. The included plugins are 
+There are some plugins bundled with the application that provide UI,
+persistence, and other default configuration which are necessary to be able to
+do anything with the application initially. Any of these plugins can, in
+principle, be replaced with a custom plugin. The included plugins are
 documented in the [Included Plugins](#included-plugins) section.  
 
+## Types
+
+The Open MCT library includes its own TypeScript declaration files which can be
+used to provide code hints and typechecking in your own Open MCT application.
+
+Open MCT's type declarations are generated via `tsc` from JSDoc-style comment
+blocks. For more information on this, [check out TypeScript's documentation](https://www.typescriptlang.org/docs/handbook/declaration-files/dts-from-js.html).
+
+### Using Types
+
+In order to use Open MCT's provided types in your own application, create a
+`jsconfig.js` at the root of your project with this minimal configuration:
+
+```json
+{
+  "compilerOptions": {
+    "baseUrl": "./",
+      "target": "es6",
+      "checkJs": true,
+      "moduleResolution": "node",
+      "paths": {
+        "openmct": ["node_modules/openmct/dist/openmct.d.ts"]
+      }
+  }
+}
+```
+
+Then, simply import and use `openmct` in your application:
+
+```js
+import openmct from "openmct";
+```
+
```

Comment:
lets make two notes:
- that we're missing some type information
- how folks can help fill in the missing pieces (if pico-mct exists, it's likely that folks would be able to help)

Reply:
Great suggestions-- filling out our types would be a perfect project to open up to the community. I'll add a section on this.